app/controllers/feedbacks_controller.rb

```diff
@@ -0,0 +1,9 @@
+class FeedbacksController < ApplicationController
```

Comment:
If edit and delete features are added I think it is better to have a separate controller.